bitcash/format.py

```diff
@@ -36,12 +45,18 @@ def verify_sig(signature, data, public_key):
 
 
 def address_to_public_key_hash(address):
-    # LEGACYADDRESSDEPRECATION
-    # FIXME: This legacy address support will be removed.
-    address = cashaddress.to_cash_address(address)
-    get_version(address)
-    Address = cashaddress.Address._cash_string(address)
-    return bytes(Address.payload)
+    if ":" not in address:
+        # Address must be a cash address, legacy no longer supported
```

Comment:
This means bitcoincash:qqqqqqqq is valid, but qqqqqqq is not. I think that's okay to me.